rspec-activemodel-mocks.gemspec

```diff
@@ -27,7 +27,7 @@ Gem::Specification.new do |s|
   s.require_path      = "lib"
 
   private_key = File.expand_path('~/.gem/rspec-gem-private_key.pem')
-  if File.exists?(private_key)
+  if File.exist?(private_key)
```

Comment:
I'm not sure this is safe for all versions of Ruby the current version of this gem professes to support?

Reply:
Made me go and look, and the docs for 1.9.3 do include https://ruby-doc.org/core-1.9.3/File.html#method-c-exist-3F